### ISSUE TITLE
invalid module name in go.mod

### ISSUE THREAD

**Issue by cboitel** [k14s/ytt#583]
**What steps did you take:**
`go get github.com/vmware-tanzu/carvel-ytt@v0`

**What happened:**
```
go get: github.com/vmware-tanzu/carvel-ytt@v0.38.0: parsing go.mod:
        module declares its path as: github.com/k14s/ytt
                but was required as: github.com/vmware-tanzu/carvel-ytt
```
**What did you expect:**
Ability to get the module or install the binary with `go install`

**Anything else you would like to add:**
Requires to update module name in go.mod. Without this change, nobody will be able to use ytt as a go library or install it simply using go install standard command.

**Environment:**

- ytt version (use `ytt --version`): N/A
- OS (e.g. from `/etc/os-release`): Ubuntu 20.04 LTS + GO 1.17.6

---
Vote on this request

This is an invitation to the community to vote on issues, to help us prioritize our backlog. Use the "smiley face" up to the right of this comment to vote.

👍 "I would like to see this addressed as soon as possible"
👎 "There are other more important things to focus on right now"

We are also happy to receive and review Pull Requests if you want to help working on this issue.

**Comment by pivotaljohn** [comment]
Carvel started out as a suite named "k14s" short for Kubernetes Tools.

Some time back, VMware formally sponsored this project and adopted it into its OSS program. The repository was moved at that time, but the module was not renamed.
We have been tracking this across the suite in vmware-tanzu/carvel#153, but haven't yet made it a priority.

Apologies for the inconvenience. Given that we're seeing more and more folks reach for a programmatic integration, I've prioritized doing this rename, soon: #585

In the meantime, feel free to rely on GitHub's redirection service:

```
go get github.com/k14s/ytt@v0
```
